lib/apiconfig/load.go

```diff
@@ -53,8 +53,8 @@ func LoadClientConfigFromBytes(b []byte) (*CalicoAPIConfig, error) {
 	if c.APIVersion != apiv2.GroupVersionCurrent {
 		return nil, errors.New("invalid config file: unknown APIVersion '" + c.APIVersion + "'")
 	}
-	if c.Kind != "calicoApiConfig" {
-		return nil, errors.New("invalid config file: expected kind 'calicoApiConfig', got '" + c.Kind + "'")
+	if c.Kind != KindCalicoAPIConfig {
```

Comment:
This changes the case of the first `c` to `C`, does that matter?

Reply:
nope - as expected as part of the v2 client.